dpnp/dpnp_iface_mathematical.py

```diff
@@ -1880,6 +1946,62 @@ def sign(
         )
 
 
+def signbit(
+    x,
+    /,
+    out=None,
+    *,
+    order="K",
+    where=True,
+    dtype=None,
+    subok=True,
+    **kwargs,
+):
+    """
+    Returns element-wise `True` where signbit is set (less than zero).
+
+    For full documentation refer to :obj:`numpy.signbit`.
+
+    Returns
+    -------
+    out : dpnp.ndarray
+    A boolean array with indication of the sign of each element of `x`.
+
+    Limitations
+    -----------
+    Parameters `x` is only supported as either :class:`dpnp.ndarray` or :class:`dpctl.tensor.usm_ndarray`.
+    Parameters `where`, `dtype` and `subok` are supported with their default values.
+    Keyword argument `kwargs` is currently unsupported.
+    Otherwise the function will be executed sequentially on CPU.
+    Input array data types are limited by supported real-valued data types.
+
+    See Also
+    --------
+    :obj:`dpnp.sign` : Returns an element-wise indication of the sign of a number.
+
+    Examples
+    --------
+    >>> import dpnp as np
+    >>> np.signbit(np.array([-1.2]))
+    array([True])
+    >>> np.signbit(np.array([1, -2.3, 2.1]))
+    array([False,  True, False])
```

Comment:
I think it would be more readable to split the lines, like:
```suggestion
    >>> import dpnp as np
    >>> x = np.array([-1.2])
    >>> np.signbit(x)
    array([True])

    >>> x = np.array([1, -2.3, 2.1])
    >>> np.signbit(x)
    array([False,  True, False])

```